tests/sqlparser_postgres.rs

```diff
@@ -662,6 +662,22 @@ fn parse_create_extension() {
         .verified_stmt("CREATE EXTENSION extension_name WITH SCHEMA schema_name VERSION version");
 }
 
+#[test]
+fn parse_drop_extension() {
+    pg_and_generic().verified_stmt("DROP EXTENSION extension_name");
```

Comment:
Since the feature introduces a new node to the ast can we add one scenario that asserts the ast in [this style](https://github.com/apache/datafusion-sqlparser-rs/blob/04d8502677a85b2c166aabedf25c6bc94a039a0b/tests/sqlparser_postgres.rs#L567-L579)?

Reply:
Done, but in a slightly different style (more succinct), and did so for all scenarios.